lib/websocket.js

```diff
@@ -1030,6 +1031,11 @@ function initAsClient(websocket, address, protocols, options) {
  */
 function emitErrorAndClose(websocket, err) {
   websocket._readyState = WebSocket.CLOSING;
+  //
+  // The following assignment is practically useless and is done only for
```

Comment:
Which summarizes the entire change pretty flawlessly. I am still grateful for this consistency! Thank you!